### PR TITLE
[Staging] [UI] Create "Featured Analysis" Section

### PR DESCRIPTION
**Pull request by kilemensi**
## Description

Implement **Featured Analysis** per design

Screenshot from 2019-03-22 17-45-00

## Type of change

- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Desktop

Screenshot from 2019-03-25 08-29-17

### Mobile

Mobile

### Interaction

Peek 2019-03-25 08-29

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: takwimu/takwimu_ui/src/Theme.js [hunk 1]
```diff
@@ -5,11 +5,25 @@ const FONT_FAMILY_TEXT = '"Muli", sans-serif';
 
 const Theme = createMuiTheme({
   palette: {
-    primary: { main: '#29a87c' }
+    primary: { main: '#29a87c' },
+    analysis: {
+      main: '#f6fbfa' // ice-blue
+    }
```
Comment (karimkawambwa) [inline takwimu/takwimu_ui/src/Theme.js]: Just make this `iceBlue: #f6fbfa`
Its used here too https://github.com/TakwimuAfrica/Takwimu/pull/471/files#diff-d4778892734c83ea1c6e5bbed582942aR9

Comment (kilemensi) [reply takwimu/takwimu_ui/src/Theme.js]: Generalized it to `palette.info.main` @karimkawambwa

**Comment by karimkawambwa** [comment]
@kilemensi 
Missed this: 🤦

Screen Shot 2019-03-26 at 9 20 05 AM

**Comment by kilemensi** [comment]
I remember @DavidLemayian said that part of the design has been/will be removed @karimkawambwa. We didn't miss it.

**Pull request merged**